data/polkit.service.in

```diff
@@ -8,7 +8,7 @@ BusName=org.freedesktop.PolicyKit1
 CapabilityBoundingSet=CAP_SETUID CAP_SETGID
 DeviceAllow=/dev/null rw
 DevicePolicy=strict
-ExecStart=@libprivdir@/polkitd --no-debug
+ExecStart=@libprivdir@/polkitd --no-debug --log-level=5
```

Comment:
The command line parameter should take string representations, rather than numbers, as it would be really confusing to use otherwise

Reply:
Yes, planned after release (that I need to make this week asap).
> as it would be really confusing to use otherwise
polkitd binary is not supposed to be run by user and the help info mentions the correct values. I guess the admins would figure out.

Reply:
It should be trivial to add? I think it would be better to fix this before the release. Admins adding a drop-in to change the default log level are the users, and it's not really nice to use numeric values like this

Reply:
It is a glib option that would probably require a callback function. I'm almost sure the result will be a few lines of code, but I've never done that and I'd have to research the docs and stuff and I really need to make that release instead.

Reply:
It doesn't need a callback, I'll prep a PR based on this and open it, probably tomorrow, including LogControl support. I'd really like to ahve both in the next release, so they can be in Debian 13

Reply:
Here it is: https://github.com/polkit-org/polkit/pull/482

The commits from this PR are squashed in the first one. The difference is using strings instead of numbers for the command line argument, and also passing the log level to the syslog() call so that messages are tagged accordingly to priority in the journal. And then another commit to add LogControl1 on top of that.

Reply:
Yeah, I implemented it with a short callback yesterday. I just didn't push it, for I didn't consider this a race.

Reply:
Also, I had presumed that implementing it with tools provided by Glib would allow having the options auto-documented in help page and/or auto-translated. Like I said, it required going through the documentation that showed that it probably doesn't cover the case of this PR (maybe I'm wrong about this, a Glib expert could know more).